packages/backend/src/storage.test.ts

```diff
@@ -824,6 +843,36 @@ describe('Storage', () => {
             );
         });
     });
+    describe('Migration', () => {
+        it('should migrate storage', async () => {
+            const { body: preMigrationStatus } = await request(app)
+                .get(`/new/bob.eth/migrationStatus`)
+                .set({
+                    authorization: `Bearer ${token}`,
+                })
+                .send();
+
+            expect(preMigrationStatus).toBe(false);
+
+            const { status } = await request(app)
+                .post(`/new/bob.eth/migrationStatus`)
+                .set({
+                    authorization: `Bearer ${token}`,
+                })
+                .send();
```

Comment:
1. I would prefer `startMigration` and `migrationStatus` or something similar in addition to `post` and `get` in order to make the difference more visible
2. Do we have any large accounts that would make finishing the migration while the call is open difficult? Then we would need a third status, e.g. `inProgress` or something similar